src/proto/world.proto

```diff
@@ -45,6 +45,11 @@ message WorldState
     optional BallState ball_state         = 3;
 }
 
+message WorldStateReceivedTrigger
+{
+    required bool world_state_received = 1;
```

Comment:
I don't think we need a boolean even here right? It seems like we don't care about the value of the boolean. So I think, we can have the same functionality by using the proto as a trigger without defining any message fields in it